tensorflow/lite/micro/examples/hello_world/evaluate.py

```diff
@@ -50,9 +50,22 @@ def invoke_tflite_interpreter(input_shape, interpreter, x_value, input_index,
 
 
 # Generate a list of 1000 random floats in the range of 0 to 2*pi.
-def generate_random_input(sample_count=1000):
+def generate_random_int8_input(sample_count=1000):
   # Generate a uniformly distributed set of random numbers in the range from
   # 0 to 2π, which covers a complete sine wave oscillation
+  np.random.seed(42)
+  x_values = np.random.uniform(low=0, high=2 * np.pi,
+                               size=sample_count).astype(np.int8)
+  # Shuffle the values to guarantee they're not in order
+  np.random.shuffle(x_values)
+  return x_values
+
+
+# Generate a list of 1000 random floats in the range of 0 to 2*pi.
+def generate_random_float_input(sample_count=1000):
+  # Generate a uniformly distributed set of random numbers in the range from
+  # 0 to 2π, which covers a complete sine wave oscillation
+  np.random.seed(42)
   x_values = np.random.uniform(low=0, high=2 * np.pi,
                                size=sample_count).astype(np.float32)
   # Shuffle the values to guarantee they're not in order
```

Comment:
let's remove the shuffle, not sure why we had this in the first place.

Reply:
done